package.json

```diff
@@ -7,10 +7,13 @@
     "prepare": "husky install",
     "build": "FEDERATE=1 webpack --mode production --config webpack.prod.js",
     "start": "DEMO_APP=true webpack serve --https --host prod.foo.redhat.com --port 1337 --hot --config webpack.dev.js",
-    "start:mocked": "DEMO_APP=true MOCKED_API=true webpack serve --hot --config webpack.dev.js",
+    "start:mocked": "DEMO_APP=true MOCKED_API=true webpack serve --hot --config webpack.dev.js --no-open",
```

Comment:
Maybe we could have both:
- `"start:mocked": "DEMO_APP=true MOCKED_API=true webpack serve --hot --config webpack.dev.js"` - for manual testing, it is useful when browser open with proper url
- `"start:mocked:it": "DEMO_APP=true MOCKED_API=true webpack serve --hot --config webpack.dev.js --no-open",` - for automated testing, when browser is nto needed

However not strong opinion, lets see hat other reviewers opinion is and of yourse, what your opinion in @sjamboro

Reply:
@jomarko I have no strong opinion. In case that my code should break something I would like to create a new JIRA and discuss / solve it separately.

I am not sure if there is some recommendation to open the browser when the application is ready. For manual testing, I will appreciate the `--no-open` option. Everytime when I restart the application then my browser window contains a new tab. It is a little bit annoying for me.

I am not sure if we should create a special script for integration tests. In case that we are able to find an agreement about setting up of the mocked application I vote for one script. A new script might be a maintenance problem.

Reply:
+1 for one script then